manifest.json

```diff
@@ -3,23 +3,20 @@
   "applications": {
     "gecko": {
       "id": "copypatch@kiszka.org",
-      "strict_min_version": "78.4.0",
-      "strict_max_version": "102.*"
+      "strict_min_version": "102.0"
```

Comment:
Why don't we need a `strict_max_version` anymore?

Reply:
This is a pure WebExtension, the Thunderbird Team makes sure it stays compatible, because we keep the APIs stable/working.